pkg/apis/security/v1alpha1/types.go

```diff
@@ -206,8 +217,11 @@ type SecurityPolicyPort struct {
 	// want match single port, you should write like 22. If you want match a range of port, you
 	// should write like 20-80, ports between 20 and 80 (include 20 and 80) will matches. If you
 	// want match multiple ports, you should write like 20,22-24,90.
-	// +kubebuilder:validation:Pattern="^(((\\d{1,5}-\\d{1,5})|(\\d{1,5})),)*((\\d{1,5}-\\d{1,5})|(\\d{1,5}))$|^$"
+	// validation:Pattern="^(((\\d{1,5}-\\d{1,5})|(\\d{1,5})),)*((\\d{1,5}-\\d{1,5})|(\\d{1,5}))$|^$"
```

Comment:
Require comment with mark `+kubebuilder:validation:` to validate this field.
https://book.kubebuilder.io/reference/markers.html#marker-syntax

Reply:
PortRange include portname, this validation is wrong. And for number PortRange, the precheck is in webhook https://github.com/everoute/everoute/blob/main/pkg/webhook/validates/crd_validate.go#:~:text=func%20(v%20*securityPolicyValidator)%20validatePortRange(portRange%20string)%20error%20%7B